assets/components/doubleHeading/doubleHeading.jsx

```diff
@@ -1,11 +1,21 @@
+// @flow
+
 // ----- Imports ----- //
 
 import React from 'react';
 
 
+// ---- Types ----- //
+
+type PropTypes = {
+  heading: string,
+  subheading: ?string,
```

Comment:
It was always meant to allow both a single heading and a double heading, so that this could be used as a more generic type of heading to cover both cases. Technically this could do the things that `simpleHeading` is currently being used for, because they're very similar.